Resources/Textures/Objects/Melee/flash.rsi/meta.json

```diff
@@ -0,0 +1,76 @@
+{
+    "version": 1,
+    "size": {
+        "x": 32,
+        "y": 32
+    },
+    "license": "GNU AGPL v3",
```

Comment:
This is a code license, you'll want the find the content license which for SS13 is usually CC BY-SA of some sort.